install/kubernetes/cilium/templates/cilium-agent/servicemonitor.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
 spec:
   selector:
     matchLabels:
-      k8s-app: cilium
+      app.kubernetes.io/name: cilium-agent
```

Comment:
Why this change? All other ServiceMonitor definitions also use "k8s-app", let's be consistent

Reply:
Sorry, I messed up the commit message while cleaning up my email. Here is the original commit message for this `Update the label to be more precise as exiting label selects both cilium-agent and hubble-peer services`

Reply:
Could you update the commit message so it's correct again? Thanks

Reply:
Done